### DISCUSSION TITLE
Simulation only with currents

### DISCUSSION THREAD

**Discussion by Alonso112358** [OpenDrift/opendrift#1329] [General]
Is there any module on open drift to run only with currents ?
I'm asking because I want to run the openoil module using only input from croco model( currents without winds)
Is it possible?

**Comment by knutfrode** [comment]
Yes, you can set winds to 0:
```
o.set_config('environment:constant:x_wind', 0)
o.set_config('environment:constant:y_wind', 0)
```

**Comment by Alonso112358** [comment]
Thanks so much
